packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -571,17 +571,25 @@ export class PerformanceEvent {
 		event: ITelemetryGenericEvent,
 		markers?: IPerformanceEventMarkers,
 		recordHeapSize: boolean = false,
+		emitLogs: boolean = true,
```

Comment:
Let's add a bit of docs to the method to clarify that even if `emitLogs = false` the class will still produce performance markers. I was going to suggest removing the param

Reply:
I will delay this until after I test it in the stress tests as I need to figure out a good narrative about what the other parameters do 😄. I can't just add documentation only for this one.